### ISSUE TITLE
CUDA out of memory when trying to train

### ISSUE THREAD

**Issue by souldyf** [geopavlakos/hamer#93]
Hello,
When I try to train the model, I get a 'CUDA out of memory' error. I have changed the TRAIN.BATCH_SIZE from 8 to 1 in hamer_vit_transformer.yaml, but it did not work. What should I do to solve this problem?
Thank you!

**Comment by geopavlakos** [comment]
Indeed, what is the GPU you are using. It's surprising you can't train even when batch size is 1.

**Comment by shitoujie** [comment]
>Hello,
When I try to train the model, I get a 'CUDA out of memory' error. I have changed the TRAIN.BATCH_SIZE from 8 to 1 in hamer_vit_transformer.yaml, but it did not work. What should I do to solve this problem?
Thank you!

me too! my gpu is 16G RTX 4060 Ti
